Source/Scene/GroundPolylinePrimitive.js

```diff
@@ -382,7 +382,7 @@ define([
             return initPromise;
         }
 
-        GroundPolylinePrimitive._initPromise = ApproximateTerrainHeights.initialize('../Assets/approximateTerrainHeights.json')
+        GroundPolylinePrimitive._initPromise = ApproximateTerrainHeights.initialize()
```

Comment:
@hpinkos  this seems to cause the JSON load to fail for local hosted and unproxied (?) apps. I'm getting a runtime error for the deployed Sandcastle demos:
http://cesium-dev.s3-website-us-east-1.amazonaws.com/cesium/fix-approx-heights/Apps/Sandcastle/index.html?src=Polyline.html

Is there a good way to test this with a proxy?

Reply:
`ApproximateTerrainHeights.initialize` needs to use `buildModuleUrl` like everything else, never load a Cesium required asset without it.

Reply:
I mean, this is what `GroundPrimitive` does already so this should be the right thing to do.  I don't know where that runtime error is coming from.  I'll see if I can figure it out.

Reply:
@mramato `ApproximateTerrainHeights.initialize` does use `buildModuleUrl`

Reply:
Ah okay, I this is happening because it's being used in a webworker for `GroundPolylinePrimitive`

Reply:
sorry about this >__<

Reply:
Probably no one has ever tried to use `buildModuleUrl` from a worker, which is pretty unusual.  I suspect an easier path would be to load the terrain height data in the main thread and send the data across, rather than trying to re-load it independently in the worker.

Reply:
Alternatively you could compute the full absolute url and send that across to the worker.